session/session.go

```diff
@@ -1509,7 +1509,8 @@ func logStmt(node ast.StmtNode, vars *variable.SessionVars) {
 		if ss, ok := node.(ast.SensitiveStmtNode); ok {
 			log.Infof("[CRUCIAL OPERATION] con:%d schema_ver:%d %s (by %s).", vars.ConnectionID, schemaVersion, ss.SecureText(), user)
```

Comment:
Do we need change this too?

Reply:
I don't think so. These are some privilege statements. The current database name is not useful for it.